ci/openstack/vars.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+if [ "${RUN_OPENSTACK_CI:-}" == true ]
+then
+    # TODO(shadower): Can we only run this when the project admin asked for it?
```

Comment:
see an example script to fetch data from github API https://github.com/kubernetes-incubator/kubespray/blob/master/scripts/premoderator.sh